### PR TITLE
Alt debseed

### LINKED ISSUE #29: Documentation : example
In the documentation for example application, under Set Up instructions, shouldn't it be `pushd example/bin/` instead of `pushd bin/`

### PR DESCRIPTION
**Pull request by heckj**
- enabling packer local or remote builds
 - updating from heckj/rackhd to rackhd/rackhd for vagrant boxes
 - setting hostname to something more sane
 - updating readme instructions to resolve #29 
 - updating CoreOS to latest version for demonstration setup

### CODE REVIEW

**Comment by heckj** [comment]
@benbp @jlongever @anhou @yyscamper @brianparry 

ready for prime time - Atlas remote builds fully operational with this setup (https://atlas.hashicorp.com/rackhd/build-configurations/rackhd), Vagrant box already loaded. README in packer/ directory updated, etc. Fixes a few misc small nits and bugs while I was in there as well.

Walked through entire demo sequence with updated vagrant box results from these builds to verify everything worked correctly.

File: packer/ansible/roles/images/tasks/main.yml [hunk 1]
```diff
@@ -49,11 +49,11 @@
 
 # hacks for install CoreOS demo - to be replaced with proxy functionality
 - name: make CoreOS directory for hosting CoreOS demonstration booting
-  file: path=/home/vagrant/src/on-http/static/http/coreos/835.8.0 state=directory
+  file: path=/home/vagrant/src/on-http/static/http/coreos/835.9.0 state=directory
```
Comment (benbp) [inline packer/ansible/roles/images/tasks/main.yml]: So this is just going to keep breaking whenever they bump it?

**Comment by benbp** [comment]
:+1:

File: packer/ansible/roles/images/tasks/main.yml (hunk 1 above)
Comment (heckj) [inline packer/ansible/roles/images/tasks/main.yml]: yeah - my path out of this hell hole is when we add the proxy support, then I can remote point to the CoreOS structures and call it "good" for a demo setup. And save a couple hundred MB on the Vagrant instance as well...

Comment (benbp) [inline packer/ansible/roles/images/tasks/main.yml]: Ah, ok

**Pull request merged**